Adyen/util.py

```diff
@@ -34,7 +34,10 @@ def escape_val(val):
     return base64.b64encode(hm.digest())
 
 
-def is_valid_hmac(dict_object, hmac_key):
+def is_valid_hmac(dict_object, hmac_key, keep_additional_data=False):
+    if keep_additional_data:
+        dict_object = dict_object.copy()
+
```

Comment:
```suggestion
def is_valid_hmac(dict_object, hmac_key):
        dict_object = dict_object.copy()
```
We decided to always keep the origin additional data.

Reply:
@AlexandrosMor I just modified the changes accordingly. can you check them?